rinja_derive/src/tests.rs

```diff
@@ -140,7 +140,7 @@ fn check_if_let() {
     // In this test, we ensure that `query` never is `self.query`.
     compare(
         "{% if let Some(query) = s && !query.is_empty() %}{{query}}{% endif %}",
-        r"if let Some(query,) = &self.s && !query.is_empty() {
+        r"if let Some(query,) = &self.s && !rinja::helpers::as_bool(&(query.is_empty())) {
```

Comment:
In such cases, I'm not sure it's an improvement since the `!` operator tells us that it's already a boolean.